pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint.go

```diff
@@ -115,27 +115,45 @@ func getEnvVar(cmd, args []string) (string, error) {
 // GetRemoteEntrypoint accepts a cache of image lookups, as well as the image
 // to look for. If the cache does not contain the image, it will lookup the
 // metadata from the images registry, and then commit that to the cache
```

Comment:
could you update this comment to say `digest` instead of `image`? (the word image above i think mostly is referring literally to the `image` variable`)